accessibility-checker-extension/webpack.config.js

```diff
@@ -118,10 +113,10 @@ module.exports = {
             chunks: ['reports']
         }),
         new HtmlWebpackPlugin({
-            template: path.join(sourceRootPath, 'html', 'UsingAC.html'),
+            template: path.join(sourceRootPath, 'html', 'usingAC.html'),
```

Comment:
@zhenwan The filename in Git has upper-case U. The build fails because the case is wrong here.

Reply:
@tombrunet you are right. in Git the file name has upper-case U, but in my local machine, the file name is lower- case U. and I have committed all my local changes......

Let me delete this file and recommit it.